native/configure.ac

```diff
@@ -303,9 +303,11 @@ AC_CHECK_HEADERS(sys/filio.h)
 AC_DEFUN([JK_CHECK_SETSOCKOPT], [
 AC_MSG_CHECKING(whether to use $1 with setsockopt())
 AC_TRY_RUN([
+#include <stdlib.h>
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <sys/time.h>
+#include <stdlib.h>
```

Comment:
No need of the second include. The one at line 306 should be enough.

Reply:
Absolutely, I'll fix shortly.